src/sentry/features/__init__.py

```diff
@@ -109,6 +109,7 @@
 default_manager.add("organizations:metric-alert-builder-aggregate", OrganizationFeature)
 default_manager.add("organizations:metrics", OrganizationFeature, True)
 default_manager.add("organizations:metrics-extraction", OrganizationFeature, True)
+default_manager.add("organizations:transaction-metrics-extraction", OrganizationFeature, True)
```

Comment:
Just want to double-check here but is intended to use flagr for this feature flag?

Reply:
yes. this featureflag is entirely temporary for now and we need to be able to turn it on and off quickly. we already have 1-2 other featureflags in projectconfig path that go to flagr.